### ISSUE TITLE
Train and validate the model

### ISSUE THREAD

**Issue by IYStoychev21** [codingburgas/2425-11-b-pp-student-practices-assignment-IYStoychev21#6]
Run training loop with different hyperparameters (learning rate, epochs)
